openshift-templates/jenkins/jenkins-persistent-template.yaml

```diff
@@ -214,3 +216,7 @@ parameters:
   displayName: Jenkins Arguments
   name: JENKINS_OPTS
   value: --sessionTimeout=0
+- description: The URL of Jenkins, to resolve the save/BUILD_URL bug
+  displayName: Jenkins URL
```

Comment:
I wondering if it is better to go and get the route by overriding the run script instead of asking for user input here. 

Thinking something like .s2i/bin/run
```
export JENKINS_URL="$(oc get route jenkins -o jsonpath={.spec.host})"

/usr/libexec/s2i/run
```

I tested it out and the configure-jenkins.groovy script will pick up this value but I haven't tested it any further at the moment.

Reply:
It is. Let's always set sensible defaults were possible, even if they need to be set programmatically. We'll need to make sure Jenkins has permissions to do that query, tho. But we should be safe there.